climpred/metrics.py

```diff
@@ -457,497 +897,784 @@ def _mae(forecast, reference, dim=None, **metric_kwargs):
+    ``e2c``, ``e2r``) and 2 when using comparisons involving individual ensemble
+    members (``m2c``, ``m2m``, ``m2r``). See
+    :py:func:`~climpred.metrics._get_norm_factor`.
 
-    Range:
-        * perfect: 0
-        * min: 0
-        * max: 1
+    .. note::
+        ``climpred`` uses a single-valued internal reference forecast for the
+        NMAE, in the terminology of Murphy 1988. I.e., we use a single
+        climatological standard deviation of the reference *within* the experimental
+        window for normalizing MAE.
 
-    See also:
-        * xskillscore.smape
+    Args:
+        forecast (xarray object): Forecast.
+        reference (xarray object): Reference (e.g. observations, control run).
+        dim (str): Dimension(s) to perform metric over. Automatically set by compute
+                   function.
+        weights (xarray object, optional): Weights to apply over dimension. Defaults to
+                                           ``None``.
+        skipna (bool, optional): If True, skip NaNs over dimension being applied to.
+                                 Defaults to ``False``.
+        comparison (str): Name comparison needed for normalization factor `fac`, see
+            :py:func:`~climpred.metrics._get_norm_factor`
+            (Handled internally by the compute functions)
+
+    Details:
+        +----------------------------+-----------+
+        | **minimum**                | 0.0       |
+        +----------------------------+-----------+
+        | **maximum**                | ∞         |
+        +----------------------------+-----------+
+        | **perfect**                | 0.0       |
+        +----------------------------+-----------+
+        | **orientation**            | negative  |
+        +----------------------------+-----------+
+        | **better than climatology**| 0.0 - 1.0 |
+        +----------------------------+-----------+
+        | **worse than climatology** | > 1.0     |
+        +----------------------------+-----------+
+
+    References:
+        * Griffies, S. M., and K. Bryan. “A Predictability Study of Simulated
+          North Atlantic Multidecadal Variability.” Climate Dynamics 13, no.
+          7–8 (August 1, 1997): 459–87. https://doi.org/10/ch4kc4.
+        * Murphy, Allan H. “Skill Scores Based on the Mean Square Error and
+          Their Relationships to the Correlation Coefficient.” Monthly Weather
+          Review 116, no. 12 (December 1, 1988): 2417–24.
+          https://doi.org/10/fc7mxd.
     """
-    weights = metric_kwargs.get('weights', None)
-    skipna = metric_kwargs.get('skipna', False)
-    return smape(forecast, reference, dim=dim, weights=weights, skipna=skipna)
+    mae_skill = __mae.function(forecast, reference, dim=dim, **metric_kwargs)
+    # TODO: Is this correct? Why take the mean here? This would take the mean over
+    # all spatial dims for instance.
+    std = reference.std(dim).mean()
```

Comment:
@aaronspring, see here. Is this supposed to be constrained over the time dimension, in case this is being applied to a grid?

Reply:
this looks odd. I suggest to remove `.mean()`